sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -922,6 +952,7 @@ static const struct platform_device_id board_ids[] = {
 					SOF_SPEAKER_AMP_PRESENT |
 					SOF_MAX98373_SPEAKER_AMP_PRESENT |
 					SOF_RT5682_SSP_AMP(1) |
+					SOF_BT_OFFLOAD_ENABLED |
```

Comment:
It might be wiser to use a macro that allows us to deal with other values, e.g. 
SOF_BT_OFFLOAD_SSP(2)

Reply:
also I am confused, you were mentioning earlier this is an ADL feature but it's added here for TGL?

Reply:
We are using TGL for development vehicle at the moment.

Reply:
> It might be wiser to use a macro that allows us to deal with other values, e.g.
> SOF_BT_OFFLOAD_SSP(2)

Will give it a try, thanks!!

Reply:
> We are using TGL for development vehicle at the moment.

There's actually no harm in defining more dailinks than what is used by the topology, so we could add this by default in TGL.
The other way around would be a mistake, we can't have a topology that refers to a non-existing dailink.

Reply:
That's great to know, will send 2nd version.